src/Sulu/Bundle/SecurityBundle/Resources/public/js/components/users/components/form/main.js

```diff
@@ -232,6 +233,16 @@ define(['app-config', 'widget-groups'], function(AppConfig, WidgetGroups) {
             ]);
         },
 
+        bindFormEvents: function() {
+            this.sandbox.dom.one(this.$el, 'focus', function() {
+                this.sandbox.dom.val('input[type="password"]', '');
+            }.bind(this), 'input[type="password"]');
+
+            this.sandbox.dom.one(this.$el, 'keyup', function() {
+                this.sandbox.dom.data('#password', 'changed', true);
```

Comment:
@nnatter do we really need to save the changed-state on the input? What about defining a placeholder in the javascript and in the end checking the input value against the placeholder to determine if they got changed